README.md

```diff
@@ -87,10 +88,6 @@ You can also achieve this by adding the detected mime type to your blacklist dir
 Links are only checked based on the extension in the url, we do not request the file at the linked url to infer the mime type.
 
 ## Requirements
-* The CSRFMiddleware needs to be placed at the bottom of the middleware
-stack. This requires to patch `ckan.config.middleware.pylons_app`. The patch is
-currently available in the data.govt.nz [CKAN repository](https://github.com/data-govt-nz/ckan/) on the `dia` branch,
-or [commit `74f78865`](https://github.com/data-govt-nz/ckan/commit/74f78865b8825c91d1dfe6b189228f4b975610a3) for cherry-pick.
```

Comment:
This patch is still needed, might need to update the the reference for flask support too.